### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: VoiceOver : MAS4.2.1: The name property of the 'Inspect' and 'Paths to roots' radio button of the 'Allocations' tab are not properly announced by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#436]
### Prerequisites

1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application."
6. Navigate to the "Choose Target" button of the Xamarin Profiler window.

### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations/Time Profiler" Profiling page.
4. Navigate to Allocations -> Allocations Tab and invoke the Tab.
5. Navigate to the Inspect/ Paths to Roots radio button on the right hand side of the page.
6. Observe the behavior of the VoiceOver for the radio buttons. 

### Expected Behavior

The VoiceOver should announce the concerned name of the radio buttons properly as displayed to the user.

### Actual Behavior

1. The VoiceOver is announcing the name of the 'Inspect' radio button as 'Settings Panel' radio button.
2. The VoiceOver is announcing the name of the 'Paths to roots' radio button as 'References Panel' radio button.
**NOTE:**
It is working fine with the 'Stack Trace' radio button.

### User Impact

The screenreader users will be misguided with the wrong name of the radio buttons announced by the VoiceOver. 

### Screenshots 

![voiceover_profilermac8 1](https://user-images.githubusercontent.com/22956130/41714846-3bb868de-756f-11e8-818c-69b7630d2f8f.png)
![voiceover_profilermac8 2](https://user-images.githubusercontent.com/22956130/41714850-42b4b958-756f-11e8-88b6-f2d1dc4e9326.png)

### Environment

macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Profiler-MAC
Wipro
